common/clustergroup/values.yaml

```diff
@@ -6,9 +6,29 @@ global:
     syncPolicy: Automatic
     installPlanApproval: Automatic
 
+# These default values are top-level, because helm does not do deep merged on values
+# files (https://github.com/helm/helm/issues/3486). So if you want to only override
+# the jobs you'd have to redefine every variable
+imperativeImage: quay.io/hybridcloudpatterns/utility-container:latest
+imperativeNamespace: "imperative"
+imperativeValuesConfigMap: "helm-values-configmap"
+imperativeCronJobName: "imperative-cronjob"
+imperativeJobName: "imperative-job"
+imperativeImagePullPolicy: Always
+imperativeActiveDeadlineSeconds: 3600
+imperativeSchedule: "*/10 * * * *"
+imperativeVerbosity: ""
+imperativeServiceAccountCreate: true
+imperativeServiceAccountName: imperative-sa
+imperativeClusterRoleName: imperative-cluster-role
+imperativeClusterRoleYaml: ""
+imperativeRoleName: imperative-role
+imperativeRoleYaml: ""
```

Comment:
I think you've gotten the wrong idea from that issue. It's only lists that have that problem. Let's discuss today

Reply:
No you're right. I initially had:
```
imperative:
  jobs:
  - name: n1
    ...
  - name: n2
    ...

  Image: image123
```

And that messed things up. I'll leave imperativeJobs under clusterGroup: and then group the main options under imperative:

Reply:
Actually let me rephrase that: if I move the "imperative:" dictionary  under clusterGroup then things do not work properly, whereas if I have a toplevel "imperative:" dictionary then things are fine.